Wikimate.php

```diff
@@ -188,6 +188,34 @@ public function login($username, $password, $domain = null)
 		return true;
 	}
 
+	/**
+	 * Gets the user agent for API requests.
+	 *
+	 * @return  string  The default user agent, or the current one defined
+	 *                  by {@link Wikimate::setUserAgent()}
+	 */
+	public function getUserAgent()
+	{
+		return $this->useragent;
+	}
+
+	/**
+	 * Sets the user agent for API requests.
+	 *
+	 * In order to use a custom user agent with all requests in the session,
```

Comment:
Just a minor nit: I think this would read a bit more clearly like this:

```suggestion
	 * In order to use a custom user agent for all requests in the session,
```

or:

```suggestion
	 * In order to use a custom user agent in all requests of the session,
```

WDYT?

Reply:
Sure, I picked the former, see f6983f7.

Btw, this comment started as a note-to-self :wink:, as in a test script against a [wiki ](https://wiki.xaseco.org/) on my personal server (so I can access the Apache logs), I found that using setUserAgent too late caused the initial token/login requests to still have the default agent string. Figured it might as well be a reminder to other library users too.